modules/developers-guide/pages/tutorials/reproducible-builds.adoc

```diff
@@ -0,0 +1,341 @@
+= Reproducible Canister Builds
+:IC: Internet Computer
+
+Thanks to its consensus protocol, the {IC} always runs canisters' code correctly.
+But this doesn't mean that it's running the _correct_ code for a canister.
+In particular, if you are a canister user as opposed to the owner, you likely want to verify that the canister is running some intended code before giving it control over your ICP tokens or confidential data.
+Verifying this requires answering two questions:
+
+. Which WebAssembly (WASM) code is being executed for a canister? 
```

Comment:
```suggestion
. Which WebAssembly (Wasm) code is being executed for a canister? 
```
My impression is that Wasm is the official contraction

Reply:
You are correct indeed, I'll do a search/replace